### PR TITLE
feat: add tooltips to headers on the LP dashboard and add some new data points

### LINKED ISSUE #1358: Other tooltips
## Story 

As a liquidity provider 
I want to understand what we mean by each of the key types of data displayed 
So that regardless of the words we use at Vega, it's understood what we mean!

## Tasks 

- [ ] Work out a description for each header / type of information, reusing what we can from console tooltip implementation on liquidity view
- [ ] Display tooltip for each type header of information (I'd see if we can use the front end component you can see here as you mouseover data in the table on the right https://explorer.vega.xyz/, screenshot attached)

## Tooltips 

Data columns

DETAIL PAGE 
- [ ] LPs - Liquidity providers
- [ ] Duration - Time in market 
- [ ] Equity-like share - The share of the markets liquidity held - the earlier you commit liquidity the greater % fees you earn
- [ ] Committed bond -  The amount of funds allocated to provide liquidity
- [ ] 24h fees - Total fees earned by the liquidity provider in the last 24 hours
- [ ] Fee level - The market's liquidity fee, or the percentage of a trade's value which is collected from the price taker for every trade

MARKET LIST
- [ ] Market (futures) - The market name and settlement asset
- [ ] Volume (24h) - The trade volume over the last 24 hours
- [ ] Committed bond - The amount of funds allocated to provide liquidity
- [ ] Status - The current market status - those below the target stake mark are most in need of liquidity

DATA ITEMS 
- [ ] Continuous: This is the standard trading mode where trades are executed whenever orders are received
- [ ] Monitoring auction-liquidity: This market is in auction until it reaches sufficient liquidity
- [ ] Opening auction: This is a new market in an opening auction to determine a fair mid-price before starting continuous trading.

image

Dashboard sketch for reference https://miro.com/app/board/uXjVOs_PBKk=/?moveToWidget=3458764531971937952&cot=14

Could do each of these as md files that are easily editable by Vega people and maybe a way to easily show or hide the (i) based on whether there is any content for it or not?

Visual styles (only) can be used from this early version of a design https://www.figma.com/file/qgCd4r61Z1r3tUkxKehhmu/Website-Design-Updates?node-id=399%3A20530

### PR DESCRIPTION
**Pull request by ciaran-**
# Related issues 🔗

Closes #1358

# Description ℹ️

Introduces Tooltips for column headings on the LP dashboard - both the Market overview/list page and Details pages, and for the `Status` data cell. 

Adds a few new datapoint columns to the market overview page - `Target Stake`, `% Target Stake Met` and `Fee Levels`.

# Demo 📺
Tooltips:

Screenshot 2022-11-17 at 15 11 53
Screenshot 2022-11-17 at 15 11 46

New columns:

Screenshot 2022-11-17 at 15 11 16

### CODE REVIEW

File: apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx [hunk 1]
```diff
@@ -90,17 +97,61 @@ export const MarketList = () => {
...
+          <AgGridColumn
+            headerName={t('% Target stake met')}
+            valueFormatter={({ data }: ValueFormatterParams) => {
+              const roundedPercentage =
+                parseInt(
+                  (data.liquidityCommitted / parseFloat(data.target)).toFixed(0)
+                ) * 100;
+              const display = Number.isNaN(roundedPercentage)
+                ? 'N/A'
+                : `${roundedPercentage}%`;
```
[the remark this answers is not in the document]
Comment (ciaran-) [reply apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx:140]: Oh great, thanks for the tip!

[the remark this answers is not in the document]
Comment (ciaran-) [reply apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx]: Thanks @MadalinaRaicu - think I've made that change, do you want to take another look?

**Review by sam-keen — CHANGES_REQUESTED** [review]

File: apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx [hunk 2]
```diff
@@ -90,17 +97,61 @@ export const MarketList = () => {
...
+              return display;
+            }}
+            headerTooltip={t(
+              'The ideal committed liquidity to operate the market.  If total commitment currently below this level then LPs can set the fee level with new commitment.'
```
[the remark this answers is not in the document]
Comment (ciaran-) [reply apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx:149]: Ah good spot - this was a placeholder which I intented to revisit. 

@ckpringle what do we want in this one (`% Target stake met` ?)

File: apps/liquidity-provision-dashboard/src/app/components/status/status.tsx [hunk 3]
```diff
@@ -33,17 +33,46 @@ export const Status = ({
...
     </div>
   );
 };
+
+const getTooltipDescription = (status: string) => {
+  let tooltipDescription = '';
+  switch (status) {
+    case MarketTradingModeMapping.TRADING_MODE_CONTINUOUS:
+      tooltipDescription =
+        'This is the standard trading mode where trades are executed whenever orders are received';
```
[the remark this answers is not in the document]
Comment (ciaran-) [reply apps/liquidity-provision-dashboard/src/app/components/status/status.tsx:63]: Thanks for the spot!

Do we have any place we typically keep the translation keys/content pairs at the moment? I've generally just been wrapping whatever the english string is with `t()` which I'm not sure is the right approach?

File: apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ciaran-) [reply apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx]: We're just gonna go with the title of the column for now. Thanks for pointing it out @sam-keen

File: apps/liquidity-provision-dashboard/src/app/components/status/status.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (sam-keen) [reply apps/liquidity-provision-dashboard/src/app/components/status/status.tsx:64]: @ciaran- Personally I tend to do a new commit to amend any requested changes, but I don't think there's a defined way that's enforced. 

As per our convo, your approach with `t()` looks like a good starting place, thanks.

**Review by sam-keen — APPROVED** [review]

**Review by MadalinaRaicu — APPROVED** [review]

**Pull request merged**